couchdb-cluster-playground/lib/couchdb_custom.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import itertools
+from collections import namedtuple
+from urllib.parse import quote, urljoin
+
+import requests
+# from checks import AgentCheck
+
+
+class ShouldRestartException(Exception):
+    pass
+
+
+class SessionContext(namedtuple("SessionContext", "host port local_port session")):
+    def request(self, path, host=None):
+        return self._request(self.port, path, host)
+
+    def local_request(self, path, host=None, url=None):
+        return self._request(self.local_port, path, host, url)
+
+    def _request(self, port, path, host, url=None):
+        url = url or "http://{}:{}".format(host or self.host, port)
+        response = self.session.get(url=urljoin(url, path))
+        response.raise_for_status()
+        return response.json()
+
+
+class CouchDBCustom():
```

Comment:
Is this class used anywhere? Maybe other parts of this module are unused as well?

Reply:
Yes that's right. I mostly copied and pasted the other file and only made modifications where needed so it didn't diverge too much.